public/pages/AnomalyCharts/containers/AnomalyOccurrenceChart.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import React from 'react';
+import { EuiBadge } from '@elastic/eui';
+import { get } from 'lodash';
+import ContentPanel from '../../../components/ContentPanel/ContentPanel';
+import { Monitor, Detector, DateRange } from '../../../models/interfaces';
+import { AnomalyDetailsChart } from './AnomalyDetailsChart';
+import { HeatmapCell } from './AnomalyHeatmapChart';
+import { filterWithHeatmapFilter } from '../../utils/anomalyResultUtils';
+import { getAnomalySummary } from '../utils/anomalyChartUtils';
+
+interface AnomalyOccurrenceChartProps {
+  onDateRangeChange(
+    startDate: number,
+    endDate: number,
+    dateRangeOption?: string
+  ): void;
+  onZoomRangeChange(startDate: number, endDate: number): void;
+  title: string;
+  anomalies: any[];
+  bucketizedAnomalies: boolean;
+  anomalySummary: any;
+  dateRange: DateRange;
+  isLoading: boolean;
+  showAlerts?: boolean;
+  anomalyGradeSeriesName: string;
+  confidenceSeriesName: string;
+  detectorId: string;
+  detectorName: string;
+  detector?: Detector;
+  detectorInterval?: number;
+  unit?: string;
+  monitor?: Monitor;
+  isHCDetector?: boolean;
+  selectedHeatmapCell?: HeatmapCell;
+}
```

Comment:
note: in the future we should look at combining these long lists of props into some common shared interfaces, since it looks like most of the chart components use a lot of the same ones. nothing to change for now but can consider it a tech debt item.

Reply:
sure. created issue for this: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/issues/316